### PR TITLE
chore: replacing npm with github packages [TOL-1700]

### PR DESCRIPTION
**Pull request by aodhagan-cf**
Migrating to GitHub packages

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -18,7 +18,9 @@ cache-key: &cache-key
 commands:
   use_npm_token:
     steps:
-      - run: echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" >> ~/.npmrc
+      - run: |
+      echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" >> ~/.npmrc
```
Comment (colomolo) [inline .circleci/config.yml:22]: Something is not right here, both indentation and registry address. Should be this?
```suggestion
        echo "//npm.pkg.github.com/:_authToken=${GITHUB_PACKAGES_READ_TOKEN}" > ~/.npmrc
```

Comment (aodhagan-cf) [reply .circleci/config.yml]: Yep silly me

Comment (colomolo) [reply .circleci/config.yml]: Indentation is still not there 🙂

**Review by colomolo — APPROVED** [review]
Sorry, I forgot to approve

**Comment by chrishelgert** [comment]
policies still contains semantic-release-ecosystem, it should be changed to semantic-release

File: .circleci/config.yml [hunk 2]
```diff
@@ -52,7 +54,7 @@ jobs:
     steps:
       - checkout
       - vault/get-secrets:
-          template-preset: semantic-release-ecosystem
+          template-preset: semantic-release
```
Comment (chrishelgert) [inline .circleci/config.yml:57]: instead of `use_npm_token` we need the `GITHUB_PACKAGES_WRITE_TOKEN`

File: .circleci/config.yml [hunk 3]
```diff
@@ -18,7 +18,9 @@ cache-key: &cache-key
 commands:
   use_npm_token:
     steps:
-      - run: echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" >> ~/.npmrc
+      - run: |
+          echo "//npm.pkg.github.com/:_authToken=${GITHUB_PACKAGES_READ_TOKEN}" > ~/.npmrc
```
Comment (aodhagan-cf) [inline .circleci/config.yml:22]: @chrishelgert do you mean we need the write token here?

**Review by MayaGillilan — APPROVED** [review]

**Review by chrishelgert — APPROVED** [review]

**Pull request merged**